Client/Frontend/Settings/ThemeSettings/ThemeMiddleware.swift

```diff
@@ -27,35 +27,23 @@ class ThemeManagerMiddleware: ThemeManagerProvider {
         switch action {
         case ThemeSettingsAction.themeSettingsDidAppear:
             let currentThemeState = self.getCurrentThemeManagerState()
-            DispatchQueue.main.async {
```

Comment:
The `DispatchQueue.main.async {` aren't needed anymore?

Reply:
No, a check was added at the Store level to send the actions from middlewares using `DispatchQueue.main.async ` already so this cleanup was needed